location_service/insert_database/insert.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import create_engine, text
+from dotenv import load_dotenv
+import pandas as pd
+import os
+
+# load environment variables
+load_dotenv()
+DB_HOST = os.getenv("DB_HOST")
+DB_NAME = os.getenv("DB_NAME")
+DB_USER = os.getenv("DB_USER")
+DB_PASSWORD = os.getenv("DB_PASSWORD")
+
+# Put path to csv file here
+df = pd.read_csv("./location_service/insert_database/sample_coordinates.csv")
+
+
+if __name__ == "__main__":
+    engine = create_engine(
+        f"mysql+pymysql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}/{DB_NAME}"
+    )
+    with engine.connect() as connection:
```

Comment:
can u add some code to check if theres data in the table. 
- if theres data in the table, give us an error saying theres pre-existing data
- but also make a function or something that allows us to override the error (so delete the pre-existing data and put our "new" data into the table)